docs/oas/business.yaml

```diff
@@ -4163,6 +4414,109 @@ components:
           timestamp: '2022-04-19 15:59:55'
           reference:
             businessId: C1234567
+    Continuation_In:
```

Comment:
Not sure if there is a naming convention here, "Consent_continuation_out" will have no capital after "_", this may not be an issue

Reply:
I think you are right. I will follow the convention we have so far. Changing it to `Continuation_in`